server/src/language.js

```diff
@@ -157,16 +160,15 @@ class ProgramCacheItem {
         const program = this.getProgram(request.textDocument.uri);
 
         if (!program) {
-            return {
-                uri: "",
-                range: {
-                    start: { line: -1, character: -1},
-                    end: { line: -1, character: -1}
-                }
-            };
+            return null;
         }
 
         const location = program.getDefinition(request.position);
+
+        if (!location) {
+            return null;
```

Comment:
This fixes a "location.uri not referenced" bug.

Reply:
Also made this in my stability branch. 👍